### PR TITLE
updates for repo and container name change

### PR DESCRIPTION
**Pull request by jmontleon**

### CODE REVIEW

File: templates/mediawiki-apb.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 apiVersion: servicecatalog.k8s.io/v1beta1
 kind: ServiceInstance
 metadata:
-  name: mediawiki123
```
Comment (rthallisey) [inline templates/mediawiki-apb.yaml]: There's a limitation in CI where the deployment/deploymentconfig needs to have the name as the service instance resource so bind creds get delivered to the right place.

This needs to be mediawiki:
https://github.com/ansibleplaybookbundle/mediawiki-apb/blob/master/roles/provision-mediawiki123-apb/tasks/main.yml#L26

File: templates/mediawiki-apb.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ kind: ServiceInstance
 metadata:
   name: mediawiki123
```
Comment (djzager) [inline templates/mediawiki-apb.yaml]: did you want to change this?

Comment (jmontleon) [reply templates/mediawiki-apb.yaml]: No, check the collapsed conversation :)

Comment (rthallisey) [reply templates/mediawiki-apb.yaml]: > There's a limitation in CI where the deployment/deploymentconfig needs to have the name as the service instance resource so bind creds get delivered to the right place.
> This needs to be mediawiki:
https://github.com/ansibleplaybookbundle/mediawiki-apb/blob/master/roles/provision-mediawiki123-apb/tasks/main.yml#L26
>
If we change it then then the resource name in the playbook needs to change too. We decided to keep it so we don't make a playbook change.

Comment (djzager) [reply templates/mediawiki-apb.yaml]: lol yuck. Makes sense to me. I'll forget tomorrow though.

**Comment by rthallisey** [comment]
ci is going to break on this. We can sort it out after we merge everything.

**Pull request merged**